File: frontend/src/dto/requests/MakePaymentRequest.js
class MakePaymentRequest {
    constructor(data = {}) {

        this.customerId =
            data.customerId || 0;

        this.loanAccountId =
            data.loanAccountId || 0;

        this.paymentAmount =
            data.paymentAmount || 0;

        this.loanName = 
              data.loanName || "";

        this.paymentDate =
            data.paymentDate ||
            new Date().toISOString();

        this.cardType = 
            data.cardType || "";

    }
}

export default MakePaymentRequest;

File: backend/AppPortal.Api/DTOs/Requests/MakePaymentRequest.cs
namespace AppPortal.Api.DTOs;

public class MakePaymentRequest
{
    public int CustomerId { get; set; }

    public int LoanAccountId { get; set; }

    public string? LoanName {get;set;}

    public decimal PaymentAmount { get; set; }

    public DateTime PaymentDate { get; set; }
}